Glyssen/MainForm.cs

```diff
@@ -172,24 +180,69 @@ private void MainForm_Load(object sender, EventArgs e)
 				LoadProject(Settings.Default.CurrentProject);		
 		}
 
-		private void LoadProject(string filePath)
+		private void InitializeProgress(int max)
+		{
+			// since there are just a few steps, double the max for smoother progress
+			m_progressBar.Maximum = max * 2;
+			m_progressBar.Value = 0;
+			m_progressBar.Visible = true;
+			m_tableLayoutPanel.Enabled = false;
+			Application.DoEvents();
+		}
+
+		private void IncrementProgress()
+		{
+			// increment 2 steps each time
+			m_progressBar.Value++;
+			Application.DoEvents();
+			m_progressBar.Value++;
+			Application.DoEvents();
+		}
+
+		private void CloseProgress()
 		{
+			m_progressBar.Visible = false;
+			m_tableLayoutPanel.Enabled = true;
+			UpdateDisplayOfProjectInfo();
+		}
+
+		private async void LoadProject(string filePath)
+		{
+			InitializeProgress(3);
+			IncrementProgress();
+			await Task.Delay(300);
```

Comment:
Do these calls actually make it take longer to load a project?  If so, would we be better off just having a marquee progress bar which goes until the load is complete?